server/bin/pbench-backup-tarballs.py

```diff
@@ -343,7 +343,7 @@ def backup_data(lb_obj, s3_obj, config, logger):
 
         # match md5sum of the tarball to its md5 file
         try:
-            archive_tar_hex_value = md5sum(tar)
+            archive_tar_hex_value = md5sum(tar)[1]
```

Comment:
I would recommend the
```
            (_, archive_tar_hex_value) = md5sum(tar)
```
idiom here, too; and in `server/bin/pbench-server-prep-shim-002.py` and `server/bin/pbench-verify-backup-tarballs.py` as well.